src/plugins/vis_type_vislib/public/vislib/components/legend/models.ts

```diff
@@ -34,62 +34,3 @@ export interface LegendItem {
 }
 
 export const CUSTOM_LEGEND_VIS_TYPES = ['heatmap', 'gauge'];
```

Comment:
Do we also want to add `heatmap` screenshot with updated legend palette?

Reply:
Sure, I'll add that - I think it renders the same.